inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/file/TextFileReader.java

```diff
@@ -79,6 +81,10 @@ public void getData() throws IOException {
                         }
                         resultLines.add(splitLines[i].trim());
                     }
+                    if (data.startsWith(splitStr, data.length() - splitStr.length() - 1)) {
```

Comment:
It's a bit difficult to understand here, can you add some explanations or examples?

Reply:
done